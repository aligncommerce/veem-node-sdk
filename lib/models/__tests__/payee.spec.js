import PayeeModel from '../payee'

describe('Payee Model', () => {
  it('should match the snapshot', () => {
    expect(PayeeModel.prototype.schema).toMatchSnapshot()
  })
})
