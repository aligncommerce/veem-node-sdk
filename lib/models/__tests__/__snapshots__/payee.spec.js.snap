// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Payee Model should match the snapshot 1`] = `
Object {
  "properties": Object {
    "businessName": Object {
      "type": "string",
    },
    "countryCode": Object {
      "type": "string",
    },
    "email": Object {
      "type": "string",
    },
    "firstName": Object {
      "type": "string",
    },
    "lastName": Object {
      "type": "string",
    },
    "phone": Object {
      "type": "string",
    },
    "type": Object {
      "enum": Array [
        "Incomplete",
        "Business",
        "Personal",
      ],
      "type": "string",
    },
  },
  "title": "Payee",
  "type": "Object",
}
`;
